Templates/devcontainer.json

```diff
@@ -18,9 +18,7 @@
 			"--ignore-missing-imports",
 			"--show-column-numbers",
 			"--no-pretty",
-			"--allow-redefinition",
-			"--install-types",
-			"--non-interactive"
```

Comment:
why not add the `--non-interactive` flag?

Reply:
For some reason, when adding this flags the mypy problems are not shown in `problems`. The only reason to use `--non-interactive` is if we want to use `--install-types` which dynamically installs the mypy required types.